Sustainsys.Saml2.AspNetCore2/Sustainsys.Saml2.AspNetCore2.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net47;net461;net6.0</TargetFrameworks>
```

Comment:
net461 target framework was changed to net462 because compiler generates warnings that dependant nuget packages have not been tested against net461 and they recommend updating to net462

Reply:
Also .net8.0 added